client/scss/utilities/_mixins.scss

```diff
@@ -127,7 +127,7 @@
   }
 }
 
-// Allow an element to be horizontally stretched outwith its container
+// Allow an element to be horizontally stretched out with its container
```

Comment:
Ah, not heard of that before, will change to 'outside of' if ok, as I didn't get that, that's what outwith meant